lib/rules/keyframe-selector-notation/index.js

```diff
@@ -148,6 +153,23 @@ function getSelectorsInBlock(atRule) {
 	return selectors;
 }
 
+/**
+ * @param {import('postcss-selector-parser').Root} selectors
+ * @returns boolean
+ */
+function isNamedTimelineRange(selectors) {
+	for (const selectorNode of Array.from(selectors.nodes)) {
+		const firstTagNode = selectorNode.nodes.filter((node) => node.type === 'tag')[0];
```

Comment:
nit

```suggestion
const [firstTagNode] = selectorNode.nodes.filter((node) => node.type === 'tag');
const firstTagValue = (firstTagNode && firstTagNode.value) || '';
```

Reply:
Thanks for reviewing. `isNamedTimelineRange()` is removed by the following suggestion:
https://github.com/stylelint/stylelint/pull/6605#discussion_r1095342096